packages/core/core/test/ParcelConfig.test.js

```diff
@@ -147,12 +147,15 @@ describe('ParcelConfig', () => {
       });
       assert(plugin);
       assert.equal(typeof plugin.transform, 'function');
+      // $FlowFixMe[method-unbinding]
       assert(logger.warn.calledOnce);
+      // $FlowFixMe[method-unbinding]
```

Comment:
I think this can be avoided by doing something like this (above):

```
let warnStub = sinon.stub(logger, 'warn');
// ... config.loadPlugin()
assert(warnStub.getCall(0).args[0], ...)
```